confidential-data-hub/docs/RESOURCES_SERVICES.md

```diff
@@ -0,0 +1,111 @@
+The Confidential Data Hub (CDH) provides resources services like retrieving a
+decryption keys and policy files from the Key Broker Service (KBS).
+
+Currently workloads can get resources via CDH from the following providers:
+
+* [CoCo KBS](https://github.com/confidential-containers/trustee/tree/main/kbs)
+* AMD SEV based on EFI secret pre-attestation (via [simple-kbs](https://github.com/confidential-containers/simple-kbs))
+
+The CDH service should be configured with one of the following Key Broker Client (KBC) that establish
+the interface with the provider:
+
+* `cc_kbc` - for CoCo KBS
+* `online_sev_kbc` - for AMD SEV with simple-kbs
+* `offline_fs_kbc` - for reading from the system's local filesystem
+
+For more information about configuring CDH, please refer to [this section]((../README.md#configuration-file)) in the [README.md](../README.md).
+
+## Usage
+
+Users are recommended to access CDH services via the [CoCo Restful API Server](../../api-server-rest/README.md), so 
+ensure that both API server and CDH process are running on the local host. By default the API server listen on
+port `8006`, whereas resources are HTTP-served from the `cdh/resource` endpoint.
+
+For example, assume that the CDH was configured with `cc_kbc`, to obtain the key `key` which is
+tagged as `1` from the `default` repository:
+
+```
+$ curl http://127.0.0.1:8006/cdh/resource/default/key/1
+```
+
+Notice that a CoCo Pod also has access to the API server as both container and CDH/API-server processes
+belong to the same network namespace. Thus, to read a resource from a Pod should be as simple as send a 
+HTTP GET request to the CDH endpoint as shown on the following deployment:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: cdh-example
+spec:
+  runtimeClassName: kata-qemu-tdx
+  containers:
+    - name: bash-curl
+      image: storytel/alpine-bash-curl:latest
+      imagePullPolicy: Always
+      command:
+        - sh
+        - -c
+        - |
+          curl http://127.0.0.1:8006/cdh/resource/default/key/1; tail -f /dev/null
+```
+
+## Example: offline_fs_kbc
+
+In this example we will use the `offline_fs_kbc` KBC to get resouces from the local filesystem.
```

Comment:
Maybe mention here that this example is not running with k8s or a VM or anything. Also minor typo `resouces` -> `resources`.

Reply:
good idea